src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -745,6 +804,7 @@ export default class DataTable extends Vue {
         ...row,
         rowNumber: rowN,
         rowId: rowId,
+        parentRowId,
```

Comment:
I think this could be avoided for now.

The method `row()` supports `rowLevel: 'parent' | 'child' | 'grandChild'`that can be passed down to `_radioButton()` method to keep track of the row level.
I would like to avoid any changes in `serializeData()` method whenever possible as its' task is too impactful.